src/uct/base/uct_iface.h

```diff
@@ -495,4 +496,24 @@ void uct_invoke_completion(uct_completion_t *comp, ucs_status_t status)
 }
 
 
+/**
+ * Calculates total length of the iov array buffers.
+ */
+static UCS_F_ALWAYS_INLINE
+size_t uct_iov_get_length(const uct_iov_t *iov, size_t iovcnt)
+{
+    size_t iov_it = 0;
+    size_t total_length = 0;
+
+    for (iov_it = 0; iov_it < iovcnt; ++iov_it) {
+        if (iov[iov_it].count > 1) {
```

Comment:
1. we don't need to add support for stride now
2. it should have been `total_length += count*length`